Add Switcher Runner S11 support (#34269)
home-assistant/home-assistant.github.io: source/_integrations/switcher_kis.markdown

- Switcher V4
- Switcher Breeze
- Switcher Runner
- Switcher Runner S11

Devices that require a token:

- Switcher Runner S11

If you completed the integration setup but are still unable to control the device, please make sure your device's firmware is up-to-date.

## Prerequisites

To enhance security, certain Switcher devices require a token for operation. In order to integrate your token-based Switcher devices with Home Assistant, you'll need the following information:

> - **The username of your Switcher Account.** You can find the username by
>   going into the Switcher app.
> - **Local control key token.** To find local control key token, browse to the
>   [Switcher GetKey API][token], enter your Switcher account username (which is an email address), and press

Reviewer: ```suggestion
- **The username of your Switcher Account**: To find the username, open the Switcher app.
- **Local control key token**: To find the local control key token, browse to the
  [Switcher GetKey API][token], enter your Switcher account username (which is an email address), and press
```

- Switcher V4
- Switcher Breeze
- Switcher Runner
- Switcher Runner S11

> Required token devices:

Reviewer 2: ```suggestion
Devices that require a token:
```

- Switcher Runner S11

If you completed the integration setup but are still unable to control the device, please make sure your device's firmware is up-to-date.

## Prerequisites

> For security reasons, some of the Switcher devices require a token to operate.
> 
> To use your Switcher token-based devices with Home Assistant,
> 
> You will need two pieces of information:

Reviewer 2: ```suggestion
To enhance security, certain Switcher devices require a token for operation. In order to integrate your token-based Switcher devices with Home Assistant, you'll need the following information:
```

- **The username of your Switcher Account.** You can find the username by
  going into the Switcher app.
- **Local control key token.** To find local control key token, browse to the
  [Switcher GetKey API][token], enter your Switcher account username (which is an email address), and press
  **Send Me The Code**.

>   The token will be sent to you by email.
>   It will look something like this: zvVvd7JxtN7CgvkD1Psujw==

Reviewer 2: ```suggestion
  The token will be sent to you by email. It will look something like this: `zvVvd7JxtN7CgvkD1Psujw==`
```
